Session.js

```diff
@@ -0,0 +1,31 @@
+class Session { 
+// To be utilized when unwrapping JSON from the future servlets. Will be on the client side and be used as a buffer  
+// for client to server information needed for other components of Virtual Movie Night.
+ 
+    constructor() {
+        this.sessionId;
+        this.instanceIp;
```

Comment:
I think in the java SessionClass (from #8), these were stored as sessionId, ipOfVM, and then screenNameOfController